src/database/postgres/mode/listdatabases.pm

```diff
@@ -1,4 +1,4 @@
-#
+    #
```

Comment:
Why this?

Reply:
I didn't add it :< (maybe a Pycharm parameter)